JsonSchema/Formats.cs

```diff
@@ -113,11 +114,27 @@ public static class Formats
 
 	static Formats()
 	{
-		_registry = new ConcurrentDictionary<string, Format>(
-			typeof(Formats)
-				.GetFields(BindingFlags.Static | BindingFlags.Public)
-				.Select(f => (Format)f.GetValue(null)!)
-				.ToDictionary(f => f.Key));
+		_registry = new ConcurrentDictionary<string, Format>(new Dictionary<string, Format>() {
```

Comment:
Can we do without instantiating the internal dictionary (just use the concurrent one)?

Reply:
ConcurrentDictionary doesn't have a constructor that is compatible with object initialization (it implements the Add method explicitly so the compiler doesn't find it). Do you have a suggestion for how to rewrite it?

Reply:
No, I guess this is fine.  A bit annoying but fine.